src/boot/efi/console.c

```diff
@@ -133,20 +132,19 @@ EFI_STATUS console_key_read(UINT64 *key, UINT64 timeout_usec) {
 
                 /* do not distinguish between left and right keys */
                 if (keydata.KeyState.KeyShiftState & EFI_SHIFT_STATE_VALID) {
+                        if (keydata.KeyState.KeyShiftState & (EFI_RIGHT_SHIFT_PRESSED|EFI_LEFT_SHIFT_PRESSED))
```

Comment:
hmm, the exprssion on the right is identical to EFI_SHIFT_PRESSED. Hence maybe just write that out like that:

```c
if (keydata.KeyState.KeyShiftState & EFI_SHIFT_PRESSED) 
        shift |= EFI_SHIFT_PRESSED;
```
and then maybe explain that this sets all of the relevant bits if any are set, because this of course will raise eyebrows...